src/registry_schemas/schemas/notice_of_withdrawal.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/notice_of_withdrawal",
+    "type": "object",
+    "definitions": {},
+    "title": "Notice of Withdrawal Filing",
+    "required": [
+        "noticeOfWithdrawal"
+    ],
+    "properties": {
+        "noticeOfWithdrawal": {
+            "type": "object",
+            "required": [
+                "business"
+            ],
```

Comment:
Hmm, I'm thinking we might need something else here. If a filing is future effective and has no incorporation number yet (for example a future effective IA), this is probably not sufficient? What would the business object look like in this

Reply:
Think `filingId` is required and `business` is optional...What do you think of it?

Reply:
Think more about it. we should use `filingId` for FED filing that we want to withdraw and use `identifier` for businesses and even for temporary businesses created for FED IAs (in this case, the identifier will start with `T`)